openpype/hosts/max/plugins/publish/validate_viewport_camera.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from openpype.pipeline import (
+    PublishValidationError,
+    OptionalPyblishPluginMixin)
+from openpype.pipeline.publish import RepairAction
+from openpype.hosts.max.api.lib import get_current_renderer
+
+from pymxs import runtime as rt
+
+
+class ValidateViewportCamera(pyblish.api.InstancePlugin,
+                             OptionalPyblishPluginMixin):
+    """Validates Viewport Camera
+
+    Check if the renderable camera in scene used as viewport
+    camera for rendering
```

Comment:
Wording is a bit unclear?

Reply:
I have edited the docstring a little bit